portfolio/src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.data;
+
+public class User {
+    private String email;
+    private String nickname = null;
+    
+    public User(String email) {
```

Comment:
Tip: It looks like we only have one usage of new User, so this constructor can be removed.